### PR TITLE
Added Important Algos

### PR DESCRIPTION
**Pull request by Ritik2604**
it contains Dijkstra, Prims, dft, bft, dfs, bfs and many more.LinkedList imp functions AVLTree simpler version DP questions

### CODE REVIEW

**Comment by Ritik2604** [comment]
Can you merge it as soon as possible why there is so much delay?

File: DataStructures/Graphs/GraphAlgos [hunk 1]
```diff
@@ -0,0 +1,437 @@
+import java.util.ArrayList;
```
Comment (deadshotsb) [inline DataStructures/Graphs/GraphAlgos:1]: Kindly provide a description for each file and a sample test (input and output) as comment lines in the beginning of the code.

Comment (Ritik2604) [reply DataStructures/Graphs/GraphAlgos:1]: Okay

**Comment by Ritik2604** [comment]
Done all the changes you told me to do

File: DataStructures/Graphs/GraphAlgos (hunk 1 above)
Comment (deadshotsb) [reply DataStructures/Graphs/GraphAlgos:51]: @Ritik2604 Good work

**Pull request merged**